modules/installation-osp-default-kuryr-deployment.adoc

```diff
@@ -53,8 +56,11 @@ account when estimating the number of security groups required for the quota.
 * Swift space requirements vary depending on the size of the bootstrap Ignition
 file and image registry.
 
-* Although the quota does not account for load balancer VM resources, they must be
-considered when deciding the OpenStack deployment size.
+* Although the quota does not account for load balancer VM resources, you must
+consider them when you decide on the {rh-openstack} deployment's size. In the
```

Comment:
On second thought:

"The quota does not account for load balancer resources, but you must consider these resources when you decide the {rh-openstack} deployment's size."